lib/services/devices/devices-NGSI-LD.js

```diff
@@ -118,7 +118,7 @@ function updateEntityHandlerNgsiLD(deviceData, updatedDevice, callback) {
             alarms.raise(constants.ORION_ALARM, error);
 
             callback(error);
-        } else if (response && response.statusCode === 204) {
+        } else if (response && (response.statusCode === 200 || response.statusCode == 204)) {
```

Comment:
Maybe a comment above this line should explain which spec version uses each "flavour".

Reply:
Same here...